wire-grpc-tests/src/test/java/com/squareup/wire/GrpcTest.kt

```diff
@@ -112,7 +110,7 @@ class GrpcTest {
     }
   }
 
-  @Test
+  @ObsoleteCoroutinesApi @Test
```

Comment:
I was sick with the warnings

Reply:
Put it on the class so you only do it once